service.ifttt/addon.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon
+	id="service.ifttt"
+	name="IFTTT Service"
+	version="1.0.0"
+	provider-name="plucka"
+	>
+	<requires>
+		<import addon="xbmc.python" version="2.26.0"/>
+	</requires>
+	<extension point="xbmc.service" library="default.py" start="login"/>
+	<extension point="xbmc.python.script" library="default.py">
+		<provides>executable</provides>
+	</extension>
+	<extension point="xbmc.addon.metadata">
+		<summary lang="en">IFTTT Service</summary>
+		<description lang="en">Calls IFTTT based on events</description>
+		<disclaimer lang="en">Open Source</disclaimer>
+		<platform>all</platform>
```

Comment:
You are missing a license tag. Please define one and fill it with the SPDX identifier of your license. See https://kodi.wiki/view/Addon.xml#.3Clicense.3E and spdx.org

Reply:
according to the license you bundle the identifier should be GPL-2.0-only or GPL-2.0-or-later - it really depends on you